### ISSUE TITLE
Push notifications to multiple channels in an array

### ISSUE THREAD

**Issue by vancooler** [adelevie/parse-ruby-client#190]
Is there any way to push notifications to multiple channels, for example channels "Fruit" and "Vegetable"? I cannot find anything in the document of this gem. 

Thanks in advance.

**Comment by rhymes** [comment]
@vancooler you are correct. That's a bug. In the meantime you can assign an array of channels by doing:

```ruby
push.channels = ['abc, 'def']

on the push object

**Comment by vancooler** [comment]
@rhymes Thanks for your help. I will try it and update here if it works.

**Comment by vancooler** [comment]
@rhymes It works! Thanks a lot. I do recommend to add it to the documentation in case someone else has similar confusions.

**Comment by torasakura** [comment]
somehow it doesn't work for me. i have
```
push = @client.push(attributes['data'])
push.channels = ['intern', attributes['channel']]
```
where `attributes['data']` looks like `"OZ: Test Titel"` and `attributes['channel']` looks like `"Kultur"`
i always get `102: equality needs a value instead of [intern Kultur]`

what am i doing wrong?

**Comment by rhymes** [comment]
@torasakura seems to me to be a protocol error.

1 - attributes['data'] has to be an hash in this form 

```ruby
{ :alert => "This is a notification from Parse" }
```

and not just a string

2 - are you sure attributes['channel'] is a string and not an object that on the console behaves as a string? Can you paste the content?

**Comment by torasakura** [comment]
@rhymes i also tried the channel names hardcoded as `push.channels = ['intern', 'Kultur']` but it didn't work either.

**Comment by torasakura** [comment]
ok i narrowed down. it seems that parse has a problem with a channels array in where. this does work
```
curl -X POST \  Di 10 Nov 2015 17:19:06 CET
    -H "X-Parse-Application-Id: application_id" \
     -H "X-Parse-REST-API-Key: api_key" \
      -H "Content-Type: application/json" \
       -d '{
          "channels": [
              "Giants",
               "Mets"
           ],
          "data": {
               "alert": "The Giants won against the Mets 2-3."
    }
 }' \
https://api.parse.com/1/push
```
this does not work:
```
curl -X POST \ Di 10 Nov 2015 17:18:44 CET
   -H "X-Parse-Application-Id: application_id" \
   -H "X-Parse-REST-API-Key: api_key" \
   -H "Content-Type: application/json" \
    -d '{
        "where": {
            "channels": ["Giants", "Mets"],
            "scores": true
        },
       "data": {
        "alert": "The Giants scored a run! The score is now 2-2."
  }
}' \
https://api.parse.com/1/push
```
and when i read push.rb right, if i use push.channels `where` is used.

**Comment by odedharth** [comment]
@toothfairy I have the same problem. Did you find a solution?

**Comment by nashby** [comment]
@odedharth @torasakura I got this issue as well and sent PR fixing it. Please check #213

**Comment by rhymes** [comment]
Closed by #213
